app/styles/theme.tsx

```diff
@@ -1,8 +1,8 @@
 "use client";
-import { createTheme } from "@mui/material/styles";
+import { createTheme, Theme } from "@mui/material/styles";
 import Link from "next/link";
 
-const theme = createTheme({
+export const themeOptions = {
```

Comment:
```suggestion
export const themeOptions: ThemeOptions = {
```
Mention type for better IntelliSense support.